packages/core/src/components/TextField/Mask.jsx

```diff
@@ -153,11 +202,15 @@ Mask.propTypes = {
  * @returns {String}
  */
 export function unmask(value, mask) {
-  if (!value) return value;
+  if (!value || typeof value !== 'string') return value;
+
+  value = value.trim();
 
   if (mask === 'currency') {
     // Preserve only digits, decimal point, or negative symbol
     value = value.match(/^-|[\d.]/g).join('');
+  } else if (['phone', 'ssn'].indexOf(mask) >= 0) {
```

Comment:
ℹ️ This currently doesn't unmask the `zip` value, though I think it should. App 3 currently expects a hyphen when it validates the value, but this could be updated or the call to `unmaskValue` could be skipped for zip code values. Thoughts, @sansumbrella @allypalanzi?

Reply:
Update: I'm going to support unmasking the zip code value in this method and consuming apps can skip calling `unmaskValue` if they rely on the value being masked, like App 3 currently. 

https://github.com/CMSgov/design-system/pull/256/commits/884739febf010904cb72cad4862f769bb24f615e